src/index.html

```diff
@@ -7,6 +7,382 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo-container">
+        <a href="#">
+          <img
+            src="./images/logo.png"
+            alt="moyo logo"
+            class="header__site-logo"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <a href="#" class="nav__link nav__link--inner-style nav__link--is-active is-active">Apple</a>
```

Comment:
why did you create extra class `nav__link--inner-style` for each link?

Reply:
> why did you create extra class `nav__link--inner-style` for each link?

Misha said, that rules which describes positioning in parent block, should be separated from "internal" rules, so I made "inner" class. May be I misunderstood this.

Reply:
in your case the rules can be written in the same class